week-5/B-callbacks/exercise.js

```diff
@@ -27,10 +28,11 @@ function exercise1() {
   Finish writing the callback function to insert the result into the #exercise2
   element
 */
-functionThatCallsBack(exercise2)
+functionThatCallsBack(exercise2);
```

Comment:
It's totally fine to add a semicolon here (and some people would argue that it's good practice). However you don't actually _need_ to add them. There are special rules where Javascript will add them for you. If you want to learn more about this try googling "automatic semicolon insertion"

Reply:
Hmm, looking at some of the other PRs, I think it's possible you have something like [Prettier](https://prettier.io/) installed. In which case feel free to ignore my comment :)

Reply:
I am using VS code and think it has the plugin installed, not sure!

Reply:
Yep I think you're right